buildspec/windowsTests.yml

```diff
@@ -0,0 +1,25 @@
+version: 0.2
+phases:
+    install:
+        commands:
+            - choco install -y --no-progress codecov
+
+    pre_build:
+        commands:
+            - npm install
+
+    build:
+        commands:
+            # max-old-space is so that v8 does not run out of memory due to webpack
```

Comment:
is the comment still relevant? `max-old-space` is absent in the rest of the script.

Reply:
oh hm I thought I removed this, removing